devmgmt/file_upload/changecaptive/templates/changecaptive/change_captive.html

```diff
@@ -164,6 +175,8 @@ <h1 align='CENTER'>Customize Captive Portal</h1>
         processData: false,
         success: function(answ){
             var data = JSON.parse(JSON.stringify(answ));
+	    $('#pinut-successAlert').show();
+		$('#pinut-infoAlert').hide();
```

Comment:
Change the sequence of show and hide..
Good idea to have hide first and show as next ?